scripts/vep_backend/vep_batch_backend.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+"""Run VEP in parallel using batch backend"""
+
+import click
+from cpg_utils import to_path
+from cpg_utils.hail_batch import dataset_path, output_path
+from cpg_utils.config import get_config
+from cpg_workflows.batch import get_batch
+from cpg_workflows.jobs.vep import add_vep_jobs
+
+
+@click.command()
+@click.option('--vcf-path', required=True, help='Path to VCF to run VEP on')
+@click.option(
+    '--output-ht',
+    required=True,
+    help='Path to where finished VEP-annotated VCF will be output',
+)
+def main(vcf_path: str, output_ht: str):
+    """
+    Run VEP in parallel using Picard tools intervals as partitions.
+    """
+    vep_image = get_config()['images']['vep']
+    b = get_batch(f'Run VEP with Batch Backend, image {vep_image}')
+    add_vep_jobs(
+        b=b,
+        vcf_path=to_path(dataset_path(vcf_path)),
```

Comment:
Thinking about this, it could be better to pass this as an explicit path?

Reply:
Yes, good point. In case the datasets are different. I'll change that